packages/engine/Source/Shaders/Voxels/IntersectBox.glsl

```diff
@@ -0,0 +1,80 @@
+// import { Ray, NO_HIT } from "./IntersectionUtils.glsl";
```

Comment:
Shouldn't these commented `import` and `export` statements in `packages/engine/Source/Shaders/Voxels/*` be removed?

Reply:
Thanks @ggetz, I replaced these with plain English comments explaining how the shader is assembled.
I do think we need some explanation there, since it is not obvious how the shader is constructed. The new comments point to the relevant JS code where the shader files are put together.